src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    private static final byte ARR_SIZE = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[ARR_SIZE];
+        for (int i = 0; i < ARR_SIZE; i++) {
+            balls[i] = lottery.getRandomBall();
```

Comment:
```suggestion
```

Reply:
What does it mean? Why I cant use balls[i]?